generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -14,6 +17,24 @@ sed -i.bak 's/canvas: None,/canvas: Some("#bevy".to_string()),/' crates/bevy_win
 # setting the asset folder root to the root url of this domain
 sed -i.bak 's/asset_folder: "assets"/asset_folder: "\/assets\/examples\/"/' crates/bevy_asset/src/lib.rs
 
+export CARGO_TARGET_DIR="target"
+
+if [ "$1" = --ci ] ; then
```

Comment:
Shouldn't this be the other way around? Optimize when in CI, but don't optimize when ran locally? Unless this is more meant as a deploy vs not deploy

Reply:
Correct, I use `--ci` here for CI vs deploy

Reply:
Ah right, so ci would be the name of the workflow and not just the general concept of CI, makes sense. I'd still suggest changing it to deploy since it feels a bit clearer to me, but I'm fine with it either way. Maybe ci-job or ci-workflow?

Reply:
Good thinking. I'll go with something more explicit though, like `--optimize-wasm`